maintain_pim/common_issues/index.rst

```diff
@@ -133,3 +133,10 @@ This parameter is set by the PIM at the index creation. If you want to apply it
                   }
               }
           }'
+
+
+A CSS update affects all the PIM pages
+--------------------------------------
+
+If you add some CSS instructions in rich text editor into product edit form and don't use specific class name for each html tag you target,
+you will update all the tags that match the selection that are present in your PIM page so if you don't want side effects on all your page be as more specific as possible in the class name you use.
```

Comment:
```suggestion
When adding CSS instructions to the rich text editor within the product edit form, avoid using generic class names. Targeting elements with non-specific classes can unintentionally style other elements on the page. To prevent unintended side effects, use highly specific class names that are unique to the desired HTML tags within your product descriptions.
```
(reworded with AI 😉)